TestingLowerBounds/KullbackLeibler.lean

```diff
@@ -561,4 +604,7 @@ lemma kl_pi_const {ι : Type*} [hι : Fintype ι] [CountablyGenerated α] [IsPro
 end Tensorization
 
 end ProbabilityTheory
--- TODO : add the chain rule, product version, then the tensorization for kl (see the one for the Renyi divergence)
+--TODO: check if the EReal are a metrizable space (I think the istance is not there, since using a lemma it says that it failed to sintethize the instance of pseudo metrizable space), if there is not, we could add it, we can metrize the EReal using the metric d(x,y) = |arctg(x)-arctg(y)|. This may be useful to apply some lemmas, for example
+--TODO: define the extended exp and log
+#check Measurable.stronglyMeasurable
+--TODO: bump mathlib, I tried to do it using `lake -R -Kenv=dev update` but it failed, giving me the error `function expected at FetchM`
```

Comment:
Let's clean those todos. They are partly outdated and we don't need them in that file on the repository.
```suggestion
```